### PR TITLE
Show discounts ui update [P0]

### PR DESCRIPTION
**Pull request by oshhh**
Updated Show Discounts page UI.
show_discounts

### CODE REVIEW

**Comment by oshhh** [comment]
@googlebot I consent.

**Comment by khushi2305** [comment]
@google I consent.

**Comment by khushi2305** [comment]
@googlebot I consent.

File: home_insurance_app/lib/pages/show_discounts.dart [hunk 1]
```diff
@@ -57,111 +61,141 @@ class _DisplayDiscountsState extends State<DisplayDiscounts> {
...
+                                      style: CustomTextStyle(
+                                          fontWeight: FontWeight.w900),
+                                    ),
+                                    onPressed: () async {
+                                      //    Get offers which the user is eligible to get after launching resource picker
+                                      List<Offer> allowedOffers =
+                                          await getAllowedOffers(context);
+                                      setState(() {
+                                        accessStructure = hasAccess();
+                                        print(hasAccess());
```
Comment (dreamplay) [inline home_insurance_app/lib/pages/show_discounts.dart:149]: nit:remove print
maybe add log statement

**Pull request merged**